core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -340,6 +340,7 @@
   MASTER_REPLICATION_COORDINATOR_THREADCHECK("master.replication.coordinator.threadcheck.time",
       "5s", PropertyType.TIMEDURATION,
       "The time between adjustments of the coordinator thread pool"),
+  @Deprecated
```

Comment:
Description should be updated to reflect when it was deprecated. I noticed while reviewing deprecated items that we hadn't been doing this for configuration properties, and it's very useful.